### PR TITLE
fix(refs T31324): Don't load DpDraggable async in DpDataTable

### PR DESCRIPTION
**Pull request by salisdemos**
When using DpTable it happend, that the DpDraggable Component could not be resolved when loaded lazy.

refs https://yaits.demos-deutschland.de/T31324

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,7 +29,7 @@
     "vue-omnibox": "^0.3.7",
     "vue-sliding-pagination": "^1.3.2",
     "vuedraggable": "^2.24.3",
-    "vuex": "^3.6.2"
+    "vuex": "^3.6.2",
```
Comment (hwiem) [inline package.json:32]:
```suggestion
    "vuex": "^3.6.2"
```

**Review by hwiem — CHANGES_REQUESTED** [review]

File: src/components/core/DpDataTable/DpDataTable.vue [hunk 2]
```diff
@@ -9,7 +10,7 @@ export default {
   name: 'DpDataTable',
 
   components: {
-    DpDraggable: () => import('../DpDraggable'),
+    DpDraggable,
```
[the remark this answers is not in the document]
Comment (salisdemos) [reply src/components/core/DpDataTable/DpDataTable.vue:13]: not sure. But we have to investigate that and find a solution hat that can work again.

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,9 @@
 
 ## UNRELEASED
 
+- Don't load DpDraggable async in DpDataTable ([@salis-demos](https://github.com/salis-demos))
```
[the remark this answers is not in the document]
Comment (salisdemos) [reply CHANGELOG.md]: 06b414266dcc9360a8f3fdbfe4b9bb95d861f619

**Review by hwiem — APPROVED** [review]
lgtm!

**Pull request merged**